src/algorithms/tonal/chordsdetection.cpp

```diff
@@ -27,7 +27,8 @@ namespace standard {
 
 const char* ChordsDetection::name = "ChordsDetection";
 const char* ChordsDetection::category = "Tonal";
-const char* ChordsDetection::description = DOC("This algorithm estimates chords given an input sequence of harmonic pitch class profiles (HPCPs). It finds the best matching major or minor triad and outputs the result as a string (e.g. A#, Bm, G#m, C). This algorithm uses the Sharp versions of each Flatted note (i.e. Bb -> A#).\n"
+const char* ChordsDetection::description = DOC("This algorithm estimates chords given an input sequence of harmonic pitch class profiles (HPCPs). It finds the best matching major or minor triad and outputs the result as a string (e.g. A#, Bm, G#m, C). The note names used as output is listed follow:\n"
```

Comment:
Some changes: 

- we need to update this example strings too: "(e.g. A#, Bm, G#m, C)" --> "(e.g. Bb, Bm, Abm, C)" 

- "The note names used as output is listed follow: ... " --> `"The following note names are used in the output: \"A\", \"Bb\", \"B\", \"C\", \"C#\", \"D\", \"Eb\", \"E\", \"F\", \"F#\", \"G\", \"Ab\"."`